bokeh/util/tests/test_forcing.py

```diff
@@ -0,0 +1,61 @@
+from numpy.testing import assert_allclose
+
+import bokeh.util.forcing as forcing
+
+def _collector(results):
+    def foo(val):
+        results.append(val)
+    return foo
+
+w = 0.3
+A = 3
+phi = 0.1
+offset = 2
+
+def test_sine():
```

Comment:
can you be more explicit in your test names e.g. test_forcing_sine_function_returns_x_when_y.

I find it helps when in the future you're not just looking at a test failure of `test_repeat`.